packages/azure-services/src/azure-queue/queue.spec.ts

```diff
@@ -196,6 +195,18 @@ describe(Queue, () => {
         });
     });
 
+    describe('getMessageCount', async () => {
+        it('getCountQueue', async () => {
+            const count = 30;
+            setupQueueGetCount(30);
+
+            const actualCount = await testSubject.getMessageCount(queue);
+            expect(actualCount).toBe(count);
+
+            verifyAll();
+        });
+    });
+
     describe('createMessage', () => {
```

Comment:
you should have test for when messageText is null, messageId is null etc to cover all branches